contracts/opsProxy/OpsProxyFactory.sol

```diff
@@ -2,23 +2,18 @@
 pragma solidity ^0.8.12;
 
 import {
-    BeaconProxy
-} from "@openzeppelin/contracts/proxy/beacon/BeaconProxy.sol";
-import {
-    Initializable
-} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-import {Proxied} from "../vendor/proxy/EIP173/Proxied.sol";
+    EIP173ProxyWithCustomReceive
+} from "../vendor/proxy/EIP173/EIP173ProxyWithCustomReceive.sol";
 import {OpsProxy} from "./OpsProxy.sol";
-import {IBeacon} from "@openzeppelin/contracts/proxy/beacon/IBeacon.sol";
 import {IOpsProxy} from "../interfaces/IOpsProxy.sol";
 import {IOpsProxyFactory} from "../interfaces/IOpsProxyFactory.sol";
 
 // solhint-disable max-states-count
-contract OpsProxyFactory is IOpsProxyFactory, IBeacon, Proxied, Initializable {
+contract OpsProxyFactory is IOpsProxyFactory {
     // solhint-disable const-name-snakecase
     uint256 public constant override version = 1;
     address public immutable ops;
-    address public override implementation;
+    address public immutable implementation;
```

Comment:
if it is immutable it means we cannot update the default implementation later?

Reply:
The OpsProxyFactory will be deployed as a proxy so it is upgradable.